cmd/gobl/keygen.go

```diff
@@ -13,6 +13,8 @@ import (
 	"github.com/invopop/gobl/dsig"
 )
 
+const defaultKeyFilename = "~/.gobl/id_es256.jwk"
```

Comment:
I noticed that `~/gobl/id_es256.jwk` and ~/gobl/id_es256.jwk.pub` were hardcoded in different places of the code, so with this an a few other changes I tried to remove that duplication.

Reply:
nice.